protos/sigstore_common.proto

```diff
@@ -108,8 +108,16 @@ enum PublicKeyDetails {
         LMS_SHA256 = 14 [deprecated = true];
         LMOTS_SHA256 = 15 [deprecated = true];
 
+        // ML-DSA
+        // USER WARNING: ML_DSA is an experimental algorithm.  In the future 
+        // it MAY be used by private Sigstore deployments, but it is not yet 
+        // fully functional.  This warning will be removed when these algorithms 
+        // are usable, but care should still be taken for production environments.
+        ML_DSA_65 = 21; // See NIST FIPS 204
```

Comment:
Can we mention these as the pure variant that takes data to sign rather than the prehash variant that takes a digest?